src/plugin.js

```diff
@@ -22,7 +23,8 @@ function setPluginOptions (pluginOptions) {
     bootstrapFilename: bootstrapFilename || 'inline.chunk.js',
     templateFilename: templateFilename || 'index.html',
     templatePath: templatePath || 'templates/[name]',
-    htmlTemplatePath: htmlTemplatePath || undefined
+    htmlTemplatePath: htmlTemplatePath || undefined,
+    htmlWebpackPluginOptions: htmlWebpackPluginOptions || {}
```

Comment:
OK, I'll add a test case for some html-webpack-plugin options maybe like favicon, minify...